test/passes.spec.js

```diff
@@ -163,3 +163,26 @@ _describe('focused fns', () => {
     env.execute();
   });
 });
+
+describe('this in fixture', () => {
+  beforeEach(function() {
+    this.field = 'value';
+  });
+
+  it('should set this correctly in the fixture', function() {
```

Comment:
Here we'd want to use the `runFns` (`runEventuallyWithPromise`, `runSync`, etc.) and check for identity with the `this` reference that the unpatched jasmine functions give us. See #6 :)